broker-daemon/state-machines/order-state-machine.js

```diff
@@ -280,7 +289,13 @@ const OrderStateMachine = StateMachine.factory({
       }
 
       const errHandler = (e) => {
-        this.reject(e)
+        // code 14 corresponds to gRPC UNAVAILABLE
+        if (e.code === 14) {
```

Comment:
totally forgot about this, but we can actually use the grpc library for this. `grpc.status.UNAVAILABLE` should make to code 14 instead of using the number.

Code would look like `if (e.code === grpc.status.UNAVAILABLE)`